.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules/
 /dist/
+/.idea/
```

Comment:
All system related ignore rules should not be added into project's .gitignore. Use system git config instead.

Reply:
Removed